.github/workflows/CD.yaml

```diff
@@ -171,7 +171,8 @@ jobs:
         run: |
           if [[ $REALDATA == true ]]; then
             echo "Retrieving real data"
-            docker run -v $(pwd)/testing/data:/testData:rw ghcr.io/d-fine/dataland/datacontainer:0.0.17
+            rm -rf ./testing/data/documents/*
```

Comment:
In principle everytime a deployment is done all existing documents are deleted and then the new ones will be uploaded, correct?

Reply:
Yes that is the idea. Reuse the existing mechanism and simply overwrite the deployed data with the "real data" content.

Reply:
Thanks for clarification